funcx/endpoint/endpoint.py

```diff
@@ -1,33 +1,57 @@
-import requests
-import argparse
+import glob
+from importlib.machinery import SourceFileLoader
+import json
 import logging
 import os
 import pathlib
-import grp
+import random
+import shutil
 import signal
+import sys
 import time
-import json
+import uuid
+
 import daemon
 import daemon.pidfile
-import lockfile
-import uuid
-import json
-import sys
-import platform
-import getpass
-import shutil
-import tarfile
-import signal
 import psutil
-import random
-import importlib.machinery
+import requests
+import texttable as tt
+import typer
+from requests import Response
 
 import funcx
-from funcx.executors.high_throughput import global_config, default_config
+from funcx.errors import *
+from funcx.executors.high_throughput import default_config as endpoint_default_config
+from funcx.executors.high_throughput import global_config as funcx_default_config
 from funcx.executors.high_throughput.interchange import Interchange
-from funcx.endpoint.list_endpoints import list_endpoints
 from funcx.sdk.client import FuncXClient
-from funcx.errors import *
+
+
+FUNCX_CONFIG_FILE_NAME = 'config.py'
```

Comment:
It could, it just wasn't currently actually being changed in anyway, and so it was a "true" global constant in a way that the attributes of the State were not